core/keyserver/epochs.go

```diff
@@ -192,11 +184,8 @@ func (s *Server) logProofs(ctx context.Context, d *domain.Domain, firstTreeSize
 		glog.Errorf("logProofs(): log.GetInclusionProof(%v, %v, %v): %v", d.LogID, epoch, secondTreeSize, err)
```

Comment:
Do you need to update the error message to reflect the new func name?

Reply:
done